chat-plugins/eShop.js

```diff
@@ -65,120 +65,88 @@ try {
 }
 
 exports.commands = {
-	//shop: 'eshop', //Uncomment this if you want this to be able to be used using the /shop command
 	eshop: {
```

Comment:
We need this command to stay as /eshop with /shop as an alias due to the fact that SG has a private shop that already uses the /shop command.